docs/installation-and-operation/serialization.md

```diff
@@ -12,116 +12,364 @@ Once you really get rolling with Metabase, it's often the case that you'll have
+
+  For example, if an exported YAML has the field `collection_id: onou5H28Wvy3kWnjxxdKQ`, then the collection `onou5H28Wvy3kWnjxxdKQ` must already exist in target instance, or there must be a YAML file with the export of a collection that has this ID.
+
+## Serialization best practices
+
+### Use the same Metabase version for source and target instance
+
+Currently, serialization only works if source and target Metabase have the same major version.
+If you are using the CLI serialization commands, the version of the .jar file that you are using to run the serialization commands should match both the source and target Metabase versions as well.
+
+### If you're using H2 as your application database, you'll need to stop Metabase before importing or exporting
 
 If you're using Postgres or MySQL as your application database, you can import and export while your Metabase is still running.
 
-## Exporting a Metabase
+### Avoid using serialization for backups
+
+Just a note: serialization is _not_ meant to back up your Metabase.
+
+See [Backing up Metabase](./backing-up-metabase-application-data.md).
+
+If you're instead looking to do a one-time migration from the default H2 database included with Metabase to a MySQL/Postgres, then use the [migration guide instead](./migrating-from-h2.md).
+
+### You'll need to manually add license tokens
+
+Metabase excludes your license token from exports, so if you're running multiple environments of Metabase Enterprise Edition, you'll need to manually add your license token to the target Metabase(s), either via the [Metabase user interface](https://www.metabase.com/docs/latest/paid-features/activating-the-enterprise-edition), or via an [environment variable](../configuring-metabase/environment-variables.md#mb_premium_embedding_token).
+
+### Metabase adds logs to exports and imports
+
+Exports: Metabase adds logs to the compressed directory as `export.log`.
+
+Imports: You can add the `-o -` flag to export logs directly into the terminal, or `-o import.log` to save to a file.
+
+## Serialization with CLI commands
 
 > To serialize data on Metabase Cloud, use the [import and export API endpoints](#serializing-metabase-via-the-api)
 
+Metabase provides [`export`](#exporting-with-cli) and [`import`](#importing-with-cli) CLI commands.
+
+See [How export works](#how-export-works), [How import works](#how-import-works), and [Serialization best practices](#best-practices) for general information about serialization.
+
+### Exporting with CLI
+
 To export the contents of a Metabase instance, change into the directory where you're running the Metabase JAR and run:
 
 ```
-java -jar metabase.jar export export_name
+java -jar metabase.jar export dir_name
 ```
 
-Where `export_name` can be whatever you want to call the directory.
+Where `dir_name` can be whatever you want to call the directory.
 
-## Export options
+### `export` options
```

Comment:
In a perfect world, these will be given in a table instead of one by one, but the area we have dedicated to an article is too narrow to have a visually appealing table